projects.go

```diff
@@ -171,6 +171,7 @@ type ContainerExpirationPolicy struct {
 	KeepN           int        `json:"keep_n"`
 	OlderThan       string     `json:"older_than"`
 	NameRegexDelete string     `json:"name_regex_delete"`
+  NameRegex       string     `json:"name_regex"`
```

Comment:
Please follow the ordering in the docs, meaning `NameRegex` should be placed above `NameRegexDelete`. And please also make sure you run `go fmt` before making a PR. Thanks!

Reply:
And maybe add `NameRegexKeep` while your at it?
<img width="791" alt="image" src="https://user-images.githubusercontent.com/4171547/206916192-200c22cc-f975-41fa-b048-800d1b5f1b30.png">